spec/features/create_object_h2_spec.rb

```diff
@@ -6,17 +6,20 @@
   let(:user_email) { "#{AuthenticationHelpers.username}@stanford.edu" }
 
   before do
-    authenticate!(start_url: "#{Settings.h2_url}/dashboard", expected_text: 'Create a new collection')
+    authenticate!(start_url: "#{Settings.h2_url}/dashboard", expected_text: 'Dashboard')
   end
 
   scenario do
+    # remove modal for deposit in progress, if present
     click_button 'No' if page.has_content?('Continue your deposit', wait: Settings.post_authentication_text_timeout)
-    click_link '+ Create a new collection'
 
-    # Checks for specific content on create collection view
+    # CREATE COLLECTION
+    expect(page).to have_content('+ Create a new collection')
```

Comment:
This is unnecessary duplication. Is there a reason for it?

Reply:
i can try without;  there was a lot of debugging going on with what came before.

Reply:
yup - it's fine without.  w00t!  Maybe it's more robust now.  I hope so.